macos/FluentUI/Button/Button.swift

```diff
@@ -497,7 +497,7 @@ class ButtonCell: NSButtonCell {
 			break
 		}
 
-		return NSRect(x: 0, y: 0, width: width, height: height)
+		return NSRect(x: 0, y: 0, width: floor(width), height: floor(height))
```

Comment:
any specific reason to apply `floor` as opposed to `ceil` for width/height?

Reply:
Yes.  The behavior of the `drawingRect` method is a bit mysterious, but the system seems to be taking the `width` and `height` values returned here, and subtracting them from 40000 to get the button size.  So I'm using `floor` on these values because that's effectively a `ceil` on the difference from 40000.